projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cds/merchandising-carousel.ts

```diff
@@ -16,9 +17,29 @@ export const DEFAULT_LANGUAGE = LANGUAGE_EN;
 export const DEFAULT_CURRENCY = CURRENCY_USD;
 const productDisplayCount = 10;
 
+export const japaneseLanguage = 'ja';
+export const englishFilmProductText = 'Film';
+export const japaneseFilmProductText = 'プ';
+export const dollarCurrencySymbol = '$';
+export const yenCurrencySymbol = '¥';
+export const canonBrandName = 'Canon';
+export const canonBrandCode = 'brand_10';
+export const canonBrandPagePath = `Brands/${canonBrandName}/c/${canonBrandCode}`;
+export const chibaStoreName = 'Chiba';
+export const filmCamerasCategoryName = 'Film Cameras';
+export const filmCamerasCategoryCode = '574';
+export const filmCamerasCategoryPagePath = `Open-Catalogue/Cameras/Film-Cameras/c/${filmCamerasCategoryCode}`;
+export const camcordersCategoryName = 'Camcorders';
+export const camcordersCategoryCode = '584';
+export const slrCategoryCode = '578';
+export const slrNonProductListCategoryPagePath = `Open-Catalogue/Cameras/Digital-Cameras/Digital-SLR/c/${slrCategoryCode}`;
+export const requestContainsConsentReference = true;
+export const checkForCarouselClickEvent = true;
```

Comment:
Don't we already have some these constants defined elsewhere? Just checking, I might be wrong.

Reply:
was in the one file, but since I separated it, I put it here as multiple files used it. But if you were wondering about other e2e, only thing that comes to mind is the currency/language-*, but it doesn't have those all those const except the symbols